modules/azure-gaming/unreal-cloud-ddc/modules/ContainerService/aks-run-command.bicep

```diff
@@ -0,0 +1,143 @@
+@description('The name of the Azure Kubernetes Service')
+param aksName string
+
+@description('The location to deploy the resources to')
+param location string
+
+@description('How the deployment script should be forced to execute')
+param forceUpdateTag  string = utcNow()
+
+@description('An array of Azure RoleIds that are required for the DeploymentScript resource')
+param rbacRolesNeeded array = [
+  'b24988ac-6180-42a0-ab88-20f7382dd24c' //Contributor
+  '7f6c6a51-bcf8-42ba-9220-52d62157d7db' //Azure Kubernetes Service RBAC Reader
+]
+
+@description('Does the Managed Identity already exists, or should be created')
+param useExistingManagedIdentity bool = false
+
+@description('Name of the Managed Identity resource')
+param managedIdentityName string = 'id-AksRunCommandProxy-${location}'
+
+@description('For an existing Managed Identity, the Subscription Id it is located in')
+param existingManagedIdentitySubId string = subscription().subscriptionId
+
+@description('For an existing Managed Identity, the Resource Group it is located in')
+param existingManagedIdentityResourceGroupName string = resourceGroup().name
+
+@description('An array of commands to run')
+param commands array
+
+@description('A delay before the script import operation starts. Primarily to allow Azure AAD Role Assignments to propagate')
+param initialScriptDelay string = '120s'
+
+@allowed([
+  'OnSuccess'
+  'OnExpiration'
+  'Always'
+])
+@description('When the script resource is cleaned up')
+param cleanupPreference string = 'OnSuccess'
+
+@description('Set to false to deploy from as an ARM template for debugging') 
+param isApp bool = true
+
+resource aks 'Microsoft.ContainerService/managedClusters@2022-01-02-preview' existing = {
+  name: aksName
+}
+
+resource newDepScriptId 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = if (!useExistingManagedIdentity) {
```

Comment:
you could bump this API version to be much newer (same with some other lines) :)